woocommerce/woocommerce-ios
# Update plan purchase error image and store creation images

## Description


The plan purchase error image in the upgrade flow had excessive empty space due to an export issue. This PR replaces that asset with one with correct empty space.

Additionally, we noticed that PDF images were significantly increasing the build size. Two decorative images in particular, on the store creation waiting screen, were 14 and 16MB respectively. We've replaced those with PNGs, which come to a total of ~210KB each, so we can assess the impact of that on the build sizes and decide which other PDF images to tackle next.

## Testing instructions


Hardcode a `.prePurchaseError(.fetchError)` in [UpgradesViewModel](https://github.com/woocommerce/woocommerce-ios/blob/ba3f05a9bcbffe2341683bfd43274c1250b2c65c/WooCommerce/Classes/ViewModels/InAppPurchases/UpgradesViewModel.swift#L118)

1. Open the app, and select a Woo Express store with a free trial
2. Tap Upgrade Now
3. Observe that the error shows with sensible empty space around the image

### Store creation
1. Create a new store
2. On the waiting screen, observe that the images are shown as previously. 

## Screenshots


![pre-purchase-error-new-image](https://github.com/woocommerce/woocommerce-ios/assets/2472348/d1e7753c-895e-4fa6-be91-99075867d954)
![store-creation-new-images](https://github.com/woocommerce/woocommerce-ios/assets/2472348/6506e472-2a8f-4a98-9b83-96c549da3d40)

---
